src/mlpack/methods/neighbor_search/CMakeLists.txt

```diff
@@ -38,5 +38,7 @@ add_cli_executable(kfn)
 add_python_binding(kfn)
 add_julia_binding(kfn)
 add_go_binding(kfn)
+add_markdown_docs(kfn "cli;python;julia;go" "geometry")
```

Comment:
I don't think we need to add this line; correct me if I'm wrong. :+1:

Reply:
Agreed, I will remove it.